src/app/pages/create-wallet/create-wallet.scss

```diff
@@ -0,0 +1,92 @@
+@import 'preact-material-components/Card/mixins';
```

Comment:
move mixins imports in `src/app/styles/_mixins.scss`

this way if you import `../../styles/base` you have all mixins available.